components/ContainerFull.tsx

```diff
@@ -1,18 +1,20 @@
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import { View, StyleSheet, Dimensions } from 'react-native';
 
-interface Props {
+type ContainerProps = {
```

Comment:
usually it's preferred to use `interface` when you describe the shape of an object (like props, Pokemon) and `type` when you e.g. want to allow only specified string values:
```ts
type Alignment = 'left' | 'right'
```
or merge interfaces/types:
```ts
type Props = MyCustomProps & TextProps;
```
but ofc the current approach works, so it's 👌